src/FTPObject.jl

```diff
@@ -1,12 +1,19 @@
 type FTP
     ctxt::ConnContext
 
-    function FTP(;host::AbstractString="", implicit::Bool=false, ssl::Bool=false,
-            verify::Bool=true, active::Bool=false, user::AbstractString="",
-            pswd::AbstractString="")
+    """
+        FTP(;hostname::AbstractString="", implicit::Bool=false, ssl::Bool=false,
+            verify_peer::Bool=true, active_mode::Bool=false, username::AbstractString="",
+            password::AbstractString="")
```

Comment:
Probably best to describe the arguments below and just use `FTP(; kwargs...) -> FTP` as the signature